sound/attributions.txt

```diff
@@ -209,3 +209,6 @@ https://freesound.org/people/C_Rogers/sounds/203368/ -- glass-shattering-hit_01.
 https://freesound.org/people/Czarcazas/sounds/330800/ -- Audio reversal/fading of Shattering Glass (Small) by Czarcazas -- https://freesound.org/s/330800/ -- License: Attribution 3.0
 
 sound/effects/bonk.ogg - recorded by oranges on a coke zero bottle, edited by ninjanomnom, released to public domain
+shell_insert2.ogg - https://freesound.org/people/CombatSFX4You/sounds/563650/ , License: CC BY-NC 3.0
+shotgun_click1.ogg - https://freesound.org/people/PNMCarrieRailfan/sounds/681694/ , License: CC BY-NC 4.0
+shotgun_fire1.ogg - https://freesound.org/people/michorvath/sounds/427595/ , License: CC0
```

Comment:
preferably put the attribution text in the same folder as the sound files, that would make it easier to find the relevant attributions in the future.